slang.h

```diff
@@ -1991,6 +1991,8 @@ extern "C"
     SLANG_API SlangReflectionTypeLayout* spReflectionTypeLayout_getPendingDataTypeLayout(SlangReflectionTypeLayout* type);
 
     SLANG_API SlangReflectionVariableLayout* spReflectionTypeLayout_getSpecializedTypePendingDataVarLayout(SlangReflectionTypeLayout* type);
+    SLANG_API SlangInt spReflectionType_getSpecializedTypeArgCount(SlangReflectionType* type);
```

Comment:
Should these be moved to be around other SlangTypeReflection_* functions instead of mixing in the SlangReflectionTypeLayout functions?

Reply:
Probably, but the C reflection API is such a messy implementation detail at this point that I wasn't inclined to bother.